wazimap_ng/profile/serializers/indicator_data_serializer.py

```diff
@@ -120,8 +134,11 @@ def IndicatorDataSerializer(profile, geography):
                                          "url": x["licence_url"]
                                      },
                                      "primary_group": x["primary_group"][0],
-                                     "groups": dataset_groups_dict[x["dataset"]]
+                                     "groups": dataset_groups_dict[x["dataset"]],
                                  },
+                                 "content": get_contet(
```

Comment:
I haven't thought about it in detail, but I'm concerned that this will result in a nested query, one per indicator, have you check performance?

Reply:
Removed this part of code